frontend/src/features/properties/components/forms/subforms/EvaluationForm.tsx

```diff
@@ -254,6 +254,8 @@ const EvaluationForm = <T extends any>(props: EvaluationProps & FormikProps<T>)
                                 formikProps={props}
                                 disabled={props.disabled}
                                 field={withNameSpace('value', type, year)}
+                                placeholder={props.disabled ? 'n/a' : ' '}
```

Comment:
Will this result in the rows with the value `' '` to not be saved to the database?

Reply:
It does not, its just an input placeholder not the value